java/cloudbuild.yaml

```diff
@@ -90,6 +90,25 @@ steps:
               "gcr.io/cloud-devrel-public-resources/java11014",
       ]
     waitFor: ["java11014-build"]
+
+    # Java 17 build
+  - name: gcr.io/cloud-builders/docker
+    args: ["build", "-t", "gcr.io/$PROJECT_ID/java17", "."]
+    dir: java/java17
+    id: java17-build
+    waitFor: ["-"]
+  - name: gcr.io/gcp-runtimes/structure_test
+    args:
+      ["-i", "gcr.io/$PROJECT_ID/java17", "--config", "java/java17.yaml", "-v"]
+    waitFor: ["java17-build"]
+  - name: gcr.io/cloud-builders/docker
+    args:
+      [
+              "tag",
+              "gcr.io/$PROJECT_ID/java17",
+              "gcr.io/cloud-devrel-public-resources/java17",
+      ]
+    waitFor: ["java17-build"]
 images:
```

Comment:
You need to add the new image here to push and tag it.

Reply:
done